src/ray/protobuf/autoscaler.proto

```diff
@@ -140,7 +140,7 @@ message NodeState {
   string instance_type_name = 11;
 
   // Observability debug string describing why the node is not idle.
-  string node_activity = 12;
+  repeated string node_activity = 12;
```

Comment:
nit
```suggestion
  repeated string node_activities= 12;
```

Reply:
Let's do this in a followup. Right now there's a convention that `[str]` is activity, and a container of `[str]` (list or dict where each entry has a [str]) is activities.